pyiron_atomistics/lammps/interactive.py

```diff
@@ -114,12 +114,18 @@ def interactive_cells_setter(self, cell):
             warnings.warn('set_relative() is deprecated as of 2020-02-26. It is not guaranteed from pyiron_atomistics vers. 0.3')
 
         if is_skewed and is_scaled:
+            self._interactive_lib_command(
+                "change_box all triclinic"
+            )
             self._interactive_lib_command(
                 "change_box all x final 0 %f y final 0 %f z final 0 %f \
                  xy final %f xz final %f yz final %f triclinic remap units box"
```

Comment:
If we are setting `change_box all triclinic` in the command above, we can probably erase `triclinic` here (same for the command below).